### DISCUSSION TITLE
Workspace search not working

### DISCUSSION THREAD

**Discussion by vibsid0986** [eclipse-theia/theia#14317] [Q&A]
Hello Team,

Workspace level search functionality is not working in docker container environment, however its working fine in development environment. After debugging I came across that the issue is due to `ripgrep ` module which was absent in 
 `node_modules/@vscode/ripgrep` in docker environment, seems like while building and downloading packages this is not downloading. Could you guide me as to what could be the problem here.

Thanks in advance,
Siddhant

**Comment by msujew** [comment]
Hey @vibsid0986,

are you using the backend bundling mechanism? Can you assert that the `rg` binary in the `lib/backend/native` directory of your app is missing?

If not, I don't see why it wouldn't work. If the download of the `rg` binary fails during the install process, the build doesn't succeed - so you wouldn't even get to a running app.
